config/package_layout.json

```diff
@@ -4,5 +4,8 @@
       "clangsa": "clang",
       "clang-tidy": "clang-tidy"
     }
-  }
+  },
+  "compiler_resource_dir" : null,
```

Comment:
I think we should be able to remove all the `compiler_resource_dir` references from the source code.
After the include path changes in #2302 it should not be used, clang should find the necessary files relative to the binary.

Reply:
Please create a separate ticket for this problem. This pull request only adds default values of some entries which are used in the analyzer/web context. If we eliminate the compiler_resource_dir we will remove it from this config file too.